model/src/main/resources/contract/AssetModuleService.wsdl

```diff
@@ -287,9 +289,71 @@
                 </xsd:complexType>
             </xsd:element>
 
-        </xsd:schema>
+            <xsd:complexType name="VesselIdentifiersHolder">
+                <xsd:sequence>
+                    <xsd:element name="cfr" type="xsd:string"/>
+                    <xsd:element name="ircs" type="xsd:string"/>
+                    <xsd:element name="iccat" type="xsd:string"/>
+                    <xsd:element name="ext_mark" type="xsd:string"/>
+                    <xsd:element name="uvi" type="xsd:string"/>
+                </xsd:sequence>
+            </xsd:complexType>
+
+            <xsd:complexType name="AssetHistGuidIdWithVesselIdentifiers">
```

Comment:
Keep only request/response types here, move the rest to their respective domain XSD. Move this type to `AssetElement.xsd`.